LeanUtils/LeanUtils/ExtractSorry.lean

```diff
@@ -0,0 +1,87 @@
+import Lean
+
+open Lean Elab Term Meta Syntax Command
+
+/-- Visit a node in the info tree and apply function `x` if the node
+is a tactic info or term info. -/
+def visitSorryNode {Out} (ctx : ContextInfo) (node : Info)
+    (x : MVarId → MetaM (Option Out)) : IO (Option Out) := do
+  match node with
+  | .ofTacticInfo i =>
+    match i.stx with
+    | `(tactic| sorry) =>
```

Comment:
Neat! The `admit` tactic can probably be supported too here (but the REPL won't be able to interact with it as it doesn't support `admit` yet).